src/main/index.ts

```diff
@@ -27,6 +27,8 @@ if (isMac && process.arch !== "arm64") {
   app.commandLine.appendSwitch("enable-features", "Metal");
 }
 
+process.env.SLIPPI_LAUNCHER = "true";
+
```

Comment:
What is this and why do we need it?

Reply:
future thing for letting dolphin know it was launched by the Launcher and that it should let the launcher handle the update process.

Reply:
pulled from this PR, will address in the future.